pandas/_libs/tslibs/tzconversion.pyi

```diff
@@ -1,25 +1,21 @@
+from __future__ import annotations
```

Comment:
maybe not needed for stub file. will need to check.

Reply:
don't we need for the `|` syntax?

Reply:
a .pyi is not parsed when running pandas. It is only used by mypy (or other type checkers). The syntax of a .pyi file is not constrained by the minimum python interpreter supported by pandas.

Reply:
> will need to check.

running pre-commit we don't have any rules that prohibit this.

Reply:
> > will need to check.
> 
> running pre-commit we don't have any rules that prohibit doing this.